src/rarity.rs

```diff
@@ -8,6 +8,10 @@ pub enum Rarity {
   Epic,
   Legendary,
   Mythic,
+  BlackUncommon,
```

Comment:
shall we keep this enum sorted in order of relative rarity? i assume black sats are less rare than Mythic